dynamic-geoassets-api/templates/index.html

```diff
@@ -0,0 +1,7 @@
+{{define "index"}}
+{{template "header"}}
+            <h2>What is Dynamic geo assets API?</h2>
```

Comment:
Might be good idea to link to the documentation? Else we risk diverging definitions.

Reply:
Not sure I understand, link to where?